### PR TITLE
Upgrade LS to 5.6.1

### PR DESCRIPTION
**Pull request by marcinc**

### CODE REVIEW

**Review by jits — APPROVED** [review]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+      - docker_password
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker login -u="ukhomeofficedigital+drone_docker_logstash" -p=${DOCKER_PASSWORD} quay.io
-      - docker tag image quay.io/ukhomeofficedigital/logstash-kubernetes:latest
-      - docker tag image quay.io/ukhomeofficedigital/logstash-kubernetes:${DRONE_COMMIT_SHA}
+      - docker login -u="ukhomeofficedigital+drone_docker_logstash" -p="$${DOCKER_PASSWORD}" quay.io
+      - docker tag logstash quay.io/ukhomeofficedigital/logstash-kubernetes:latest
```
Comment (vaijab) [inline .drone.yml]: Maybe use a specific tag here too, since docker daemon is somewhat shared?

**Review by vaijab — COMMENTED** [review]
Minor comment, otherwise LGTM.

**Comment by vaijab** [comment]
LGTM!

**Pull request merged**